rdkcentral/Ripple
# Commit privacy settings to local on success from ESS

## What

Currently there is a long delay when a user selects a toggle in the privacy settings app as Ripple waits for the linchpin notification, like 4 seconds. The user is waiting a this time to continue.

## Why

To try and get this delay down, Ripple should not need to wait for linchpin in this case.

## How

if the call to ESS results in 200 status code then it is successful and Ripple should update its local cache. Change it so on success for the set to ESS:

- Persist to local store/cache
- Send out the onChanged firebolt notification for that setting (if it changed)

## Test
- Verify that privacy settings is updated to ESS
- Verify that privacy setting is updated locally
- Verify subscriber receives privacy setting changes notification 
- Verify linchpin notification is received
- Check no local update, no notification is sent after receive linchpin notification.

## Checklist

- [ X] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective
